javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,166 @@
+/**freecodecamp.org username qamarfj */
+
+/**Flight booking fullname function */
+function getFullname(firstname, surname, useFormalName = false) {
+    //  are firstname and surname are not empty and there values are not  false or true? 
+    if (arguments.length >= 2 && arguments[1] !== true && arguments[1] !== false) {
```

Comment:
Thank you for reviewing my home work.
Here I am checking there is valid firstname and surname. First condition "arguments.length >= 2" checks there is at least 2 arguments and  " arguments[1] !== true && arguments[1] !== false" checks  the second argument is text for  surname and not Boolean value for useFormaleName.
i.e. if we call function like getFullname('viktor', true)  then it will ask for valid first name and surname.

Reply:
I got it , so if you writing "arguments[1] !== true && arguments[1] !== false" for checking type of arguments[1]. try this one
typeof arguments[1] === 'string'

Reply:
Thank you I have change to typeof arguments[1] === 'string'